frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/GitMacheteGraphTable.java

```diff
@@ -258,7 +261,7 @@ public void mouseClicked(MouseEvent e) {
         return;
       }
 
-      selectedBranchName = graphItem.getValue();
+      selectedBranchName = graphItem.asBranchItem().getBranch().getName();
```

Comment:
Unrel: I didn't like the assumption that `value` of this item is always gonna be just exactly the branch name... unlikely to be broken but still kinda brittle